dist/backdraft.js

```diff
@@ -532,6 +532,16 @@ _.extend(Plugin.factory, {
       return !columnConfig.bulk;
     });
 
+    // read initial filters from URL
+    var urlParamString = window.location.href.split("?")[1];
+    if (urlParamString && $.deparam(urlParamString) && $.deparam(urlParamString).ext_filter_json ) {
```

Comment:
I thought we were going to rename the ext_filter_json param so it wasn't called EXT anymore. This was to prevent future developer confusion.